WooCommerce/Classes/POS/Models/PointOfSaleSettingsController.swift

```diff
@@ -0,0 +1,137 @@
+import Foundation
+import struct Yosemite.SiteSetting
+import enum Yosemite.Plugin
+import class Yosemite.PluginsService
+import Observation
+
+import protocol Yosemite.PointOfSaleSettingsServiceProtocol
+import class Yosemite.PointOfSaleSettingsService
+import Storage
+
+protocol PointOfSaleSettingsControllerProtocol {
+    var receiptStoreName: String? { get }
+    var receiptStoreAddress: String? { get }
+    var receiptStorePhone: String? { get }
+    var receiptStoreEmail: String? { get }
+    var receiptRefundReturnsPolicy: String? { get }
```

Comment:
Most likely I will end grouping these into a separate struct, just left it out of the scope of the PR for now.